### PR TITLE
Docs/contributors  adding projects

### PR DESCRIPTION
**Pull request by cereallarceny**
Attempting to figure out how we may better guide users from "I want to help on this project" to actually joining a specific team or part of the OpenMined ecosystem.  How do we direct people where they need to go to start helping?... rather than assume they'll figure it out on their own by reading Slack.

[See Issue 36.](https://github.com/OpenMined/Docs/issues/36)

### CODE REVIEW

**Comment by cereallarceny** [comment]
I've gone ahead and filled out "Machine Learning" as a potential example... we'll want to document the others.

File: contributing/quickstart.md
Comment (anoff) [inline contributing/quickstart.md]: Step 3 is there twice. I assume the first one is legacy? :)

File: contributing/quickstart.md
Comment (anoff) [inline contributing/quickstart.md]: Programming language and domain seem to be non-exclusive. Do you think putting the whole thing in a table would be possible?

for example:

| # | python | nodeJS | react |
|:----------------------:|----------------|--------------------|----------|
| blockchain | pySonar | mine.js capsule.js | - |
| homomorphic encryption | pySyft | capsule.js | - |
| UX | sonar notebook | adapters | sonar UI |

File: contributing/quickstart.md
Comment (anoff) [inline contributing/quickstart.md]: it's gonna be `capsule.js` 👯

**Comment by kevinahuber** [comment]
I would leave this out of the quickstart, and make it a link outside of it. I think its SUPER overwhelming to have the project as a whole, and the quickstart is like the tutorial level to OpenMined. You kill some skeletons, make a fire, do your first spell, _then_ receive a map from uncle Patrick upon completion.

**Comment by cereallarceny** [comment]
Fair enough, that's not a bad idea.  Maybe treat it as a "token of accomplishment"?  The only problem I see with that is it requires people to have a knowledge of Python and machine learning.  Most front-end developers, much less designers, UX specialists, lawyers, and marketers will be able to get past that though.  I would be concerned that could be a massive bottleneck for less technical individuals.

**Comment by cereallarceny** [comment]
@iamtrask I added the other sections in, but I'm not sure if I got all the right repos in the right place.  Can I get a quick copy check to make sure all the information is factual?  Also note, we don't have a sonar.js library so that's missing a link until further notice.

**Comment by cereallarceny** [comment]
@swaroopch I implemented the changes to the Slack channel names as discussed.  Would you mind confirming this in my latest commit?

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: typo - `s/Eterheum/Ethereum`

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: typo - s/Eterheum/Ethereum

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: Remove the link for now?

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: Remove `#terminology` - no longer exists

**Review by swaroopch — COMMENTED** [review]
LGTM!

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: say "all" instead?

**Comment by swaroopch** [comment]
@cereallarceny Do we want to mention `#beginner` and `#questions` channels?

**Comment by cereallarceny** [comment]
Good idea, definitely.  Working on a commit for all these changes.

**Comment by cereallarceny** [comment]
Approve the review and merge me in boss!

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: extra leading square bracket??

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: extra leading square bracket??

File: contributing/quickstart.md
Comment (swaroopch) [inline contributing/quickstart.md]: I believe it is called `#general-discussion` and not `#general`?

**Comment by swaroopch** [comment]
> Approve the review and merge me in boss!

I can do that!? 😮

Just one blocking change - about `#general-discussion` (not `#general`) and we should be good to merge 🙂

**Comment by cereallarceny** [comment]
Pushed up those changes, sir!  I think you can review it if you want.  We can't merge until a review has been approved.  :)

**Review by swaroopch — APPROVED** [review]

**Comment by swaroopch** [comment]
@cereallarceny github says "At least one approved review is required by reviewers with write access." - I guess I don't have write access :)

**Comment by cereallarceny** [comment]
Dang it!  @iamtrask We need your help!

**Review by anoff — APPROVED** [review]
me to the rescue \o/

**Review by iamtrask — CHANGES_REQUESTED** [review]
I second the notion that this is very intimidating as a document. Step 3 should instead be the last step, and it should broken out into sub pages (sortof like the old Goosebumps books!...choose your own ending).

**Comment by cereallarceny** [comment]
Why do we want to put these at the end @iamtrask?  What about for people who aren't technical or aren't Python developers?  I can't imagine them being very interested in spinning up Sonar if they only have UI design expertise.  I think that narrows the scope of interested individuals significantly -
 maybe we should appeal to less technically savvy individuals as well?

**Review by kevinahuber — CHANGES_REQUESTED** [review]
@cereallarceny brings up a good point, but I don't think the answer is to put this part in the middle. 
This quickstart is best for particularly python developers and the first components we had code for. After Hydrogen, the quickstart  _should_ be as simple as installing docker and running `docker-compose up`. BUT, we are not there yet and I think this is a good add to merge soon.

The solution I would like best (and what I think meets most sentiments):
- [ ] Move Step 3 to the final Step
- [ ] Include a note at the end of Step 2 that reads(ish):

> Great! You've made it this far. Steps 3-6 get pretty weedsy into code. If you are not a developer and would still love to contribute, hop down to [Step 7](#whooop-this-isnt-a-real-link)

Then in the future Steps 3-6 will be increasingly non-dev friendly. Thoughts?

@iamtrask What are your goals for this document?

**Comment by cereallarceny** [comment]
I think that's a great idea @kevinahuber - seems like a good balance.  I'll implement this now.

**Review by kevinahuber — APPROVED** [review]
👍

**Review by iamtrask — APPROVED** [review]
beautiful!

**Pull request merged**
